server/controllers/listing.js

```diff
@@ -26,13 +26,13 @@ class ListingController {
     }
   };
 
-  getOneListing = async (req, res, next) => {
+  getListing = async (req, res, next) => {
     try{
       const listingId = req.params.id;
-      const listing = await this.listingService.getOneListing(req.params.id);
+      const listing = await this.listingService.getListing(req.params.id);
       if(listing == null){
         next(ApiError.notFound(`Listing with id ${listingId} is not found.`));
-        return; 
+        return;
```

Comment:
We have started to remove these ```return```s in the controllers. After a ```next()```. I mentioned this in my last review too btw 😛  But just this one line change and that's it!

Reply:
Yeah, I'm facing a problem though! If I remove this, the error message doesn't print properly

Reply:
![withoutReturn](https://user-images.githubusercontent.com/56322456/108923337-408c1f80-75fe-11eb-80cf-13e493f28bbc.JPG)

Reply:
Think you have to zoom a little bit to see the error.

Reply:
Ah I see. I think the logic order may be the issue then. Just leave it in for now then.

Reply:
There is still a merge conflict tho, so lemme know if you need help resolving that!